ApplicationRequirements.txt

```diff
@@ -3,13 +3,28 @@
 	"Password" Text Box
 	"Sign-in" Button
 		-> Takes to "Home" Screen if correct
+			-> Takes to the Driver variant of the "Home" Screen if account is registered to a driver
 		-> Redirects back to "Log-in" Screen if incorrect
 			-> Notifies user with red text
 	"Create New Account" Button
 		-> Takes to "Create New Account" Screen
 		
-"Home" Screen
-
+"Home" Screen - General
+	"Create New Request" Button
+		-> Takes to "Create New Request" Screen
+	"My Requests" Button
+		-> Takes to "My Requests" Screen
+		
+"Home" Screen - Driver
+	"Create New Request" Button
```

Comment:
Why can the driver create new requests?

Reply:
I was going back and forth on this. I finally figured that it would kind of suck to be a driver and then not be able to use the service for yourself. I'd like to think Uber drivers can still use the Uber app like a normal user but I'm not entirely sure.

Reply:
We were talking about this in class. I thought we agreen on having the drivers logout from their driver account and login as a general user. But I do like this idea more.